### PR TITLE
Browser tests for /events/ page

### PR DESCRIPTION
**Pull request by dpford**
This adds some tests for the /events/ page, including some fake data to populate the test ES index with.

To test it, make sure you've setup the webdriver as outlined in the [README](https://github.com/cfpb/cfgov-refresh#how-to-test-the-software), then:

`cd _tests/browser_testing/`
`behave --tags=events`

It looks like some recent work to Newsroom has broken a few of those existing tests, so this command will only run the tests tagged with `@events`.

As @willbarton pointed out, some of this will need to change (specifically the fixture data schema and the expected outputs from the feature file) once real event data is coming out of WP.  

Review:
@Scotchester 
@willbarton 
@kurtw

### CODE REVIEW

File: _tests/browser_testing/fixtures/events.json [hunk 1]
```diff
@@ -0,0 +1,1171 @@
...
+        "post_count": 18,
+        "id": 1331,
+        "description": "",
+        "slug": "delicia-hand",
+        "title": "Delicia Hand"
+      }
+    ],
+    "order": 0
+  }
+]
```
Comment (willbarton) [inline _tests/browser_testing/fixtures/events.json]: My only immediate comment is that this isn't what the event JSON will look like. My (closed ATM) PR #439 has some sample JSON in it.

The front end stuff will have to change for that PR anyway, so I don't think this should block this from being merged. But the templates and tests will need to be modified when that PR is opened again and merged.

Comment (dpford) [inline _tests/browser_testing/fixtures/events.json]: Ah, thanks Will, I forgot to mention that I expect to have to change both our fixture data and the expected output in the feature file after we get actual event JSON.  I'm not convinced this necessarily needs to get merged before that happens if it's going to be so soon.  Swapping those out should be a very quick change.

**Comment by jimmynotjim** [comment]
Does this need to be updated due changes in #439?

**Comment by dpford** [comment]
Yep.  We have a jira task for it

**Comment by jimmynotjim** [comment]
Ok, wasn't sure if I should be giving a :+1: yet.

**Comment by jimmynotjim** [comment]
I think we can close this one since we've replaced the browser tests (unless we shouldn't have removed the original browser tests?)

**Comment by dpford** [comment]
Yep, agreed.

**Pull request closed without merging**